scripts/plot_light_curve.py

```diff
@@ -1,17 +1,20 @@
 from argparse import ArgumentParser
 
 import numpy as np
+from astropy.table import Table
+from astropy.time import Time
 from gammapy.estimators import FluxPoints
 from matplotlib import pyplot as plt
 from matplotlib.dates import ConciseDateFormatter
 
 parser = ArgumentParser()
 parser.add_argument("-i", "--input-path", required=True)
 parser.add_argument("-o", "--output", required=True)
+parser.add_argument("--blocks", required=False)
```

Comment:
`default=None`, ~~`reuired=False`~~

Reply:
Any reason for that? Looks the same to me

Reply:
Ah yes. I first thought this was a boolean flag, and edited later. The `required` is not needed, as arguments starting with two dashes are by default not required. The `default=None` is indeed not needed.